src/admin.ts

```diff
@@ -1,10 +1,46 @@
 import axios from "axios";
-import { Runnable } from "./types/runnable";
+import { Runnable, VersionedRunnable } from "./types/runnable";
 
 interface AdminConfig {
   baseUrl?: string;
 }
 
+interface AvailableFunctions {
+  functions: Array<SingleAvailableFunction>;
```

Comment:
```suggestion
  functions: SingleAvailableFunction[];
```
This way of writing it is a bit more idiomatic!